docs/upgrade/3.1.rst

```diff
@@ -44,3 +44,13 @@ Migrations packages has been renamed to the standard layout:
 
 South 1.0.2 is now required to correctly handle the above layout; if you are upgrading from Django 1.7 / django CMS 3.0.x,
 please remove the old migration path from ``MIGRATION_MODULES`` settings.
+
+Structure mode permission
+=========================
+
+The new "**Can use Structure mode**" permission has been added. Without this permission, a
+non-superuser cannot enter structure mode anymore: this allow more strict workflows which only
+enable content update for certain users.
+This will require that **any** non-superuser which is supposed to be allowed to use
+structure mode, must be given the new permission.
+
```

Comment:
Its great that we have this note here, but shouldn't we, by default, turn this new permission on for existing Staff users?